packages/identity-app/src/Identity.tsx

```diff
@@ -96,9 +100,53 @@ export class Identity extends React.PureComponent<Props, State> {
                 </Stacked>
               </WalletCard>
             </Grid.Column>
+
+            <Grid.Column>
+              <WalletCard
+                header='Saved Accounts'
+                subheader='To quickly move between accounts, select from the list of unlocked accounts below.'
+                overflow='scroll'>
+                <Stacked>
+                  <WithSpace>
+                    <React.Fragment>
+                      {
+                        this.renderAllAccountsFromKeyring()
+                      }
+                    </React.Fragment>
+                  </WithSpace>
+                </Stacked>
+              </WalletCard>
+            </Grid.Column>
           </Grid.Row>
         </Grid>
       </Container>
     );
   }
+
+  renderAllAccountsFromKeyring () {
+    // shouldn't load keyring more than once per session
```

Comment:
I think keyring.loadAll() should be called only once, and inside light-apps, after the api has been set. I'll do my next PR on integrating the api, you can put a FIXME here, and I'll change here as well.